wasm-node/Cargo.toml

```diff
@@ -0,0 +1,29 @@
+[package]
+name = "wasm-node"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+celestia-node = { workspace = true, features = ["test-utils"] }
+celestia-types = { workspace = true, features = ["test-utils"] }
+
+anyhow = "1.0.71"
+console_error_panic_hook = "0.1.7"
+futures-timer = { version = "3.0.2", features = ["wasm-bindgen"] }
+js-sys = "0.3.64"
+libp2p = "0.52.3"
+serde = "1.0.189"
+serde-wasm-bindgen = "0.6.0"
+serde_repr = "0.1"
+tracing = "0.1.37"
+tracing-wasm = "0.2.1"
+wasm-bindgen = "0.2.87"
+wasm-bindgen-futures = "0.4.37"
+web-sys = { version = "0.3.64", features = ["console"] }
+serde_json = "1.0.107"
```

Comment:
we should have ci check for that :+1:

Reply:
Thanks for noting that!
For tools, I've tried [`udep`](https://lib.rs/cargo-udeps) and discovered [`machete`](https://github.com/bnjbvr/cargo-machete) this morning. Former doesn't seem to flag dependencies we could remove but that are used by our dependencies though. Second one gave me more accurate results and has convenient option for flagging false positives (which they both have..).
Created https://github.com/eigerco/celestia-node-rs/issues/118 to add CI check

Reply:
Ah, actually - I've added the `futures-timer` not directly for our sake, but because it's used in `libp2p_autonat::behaviour`, but libp2p doesn't set `wasm-bindgen` feature. This causes futures-timer to use `std::time`, which panics in browser.

Reply:
that sounds like an issue in libp2p, you could go for the first contribution :)

Reply:
actually nr2: looks like libp2p has `wasm-bindgen` feature that should be set so that it can forward it to its dependencies. Added that and removed `futures-timer` 👍